consensus/src/threshold_simplex/actors/resolver/actor.rs

```diff
@@ -502,56 +515,44 @@ impl<
                             };
                             self.inflight.clear(request.id);
 
-                            // Parse notarizations
+                            // Verify message
+                            if !response.verify(&self.namespace, &identity) {
```

Comment:
We now perform an aggregate verification over the response (rather than one at a time).